lib/Dialect/CGGI/IR/CGGIOps.td

```diff
@@ -114,6 +114,20 @@ def CGGI_Lut3Op : CGGI_LutOp<"lut3", [AllTypesMatch<["a", "b", "c", "output"]>]>
   let hasCanonicalizeMethod = 1;
 }
 
+def CGGI_PackedLut3Op : CGGI_Op<"packed_lut3", [
+    Pure,
+    SameOperandsAndResultType
```

Comment:
Just for consistency, can you add `DeclareOpInterfaceMethods<LUTOpInterface>`? We haven't implemented any interesting things in this interface but the rest of our LUT ops in this file have it.

(I guess in the future it out be nice to replace all our Lut3-specific passes with the LUTOpInterface instead)

Reply:
Done!